proto/buf/alpha/registry/v1alpha1/repository_commit.proto

```diff
@@ -61,6 +63,10 @@ service RepositoryCommitService {
   // GetRepositoryCommitBySequenceId returns the repository commit matching
   // the provided sequence ID and branch, if it exists.
   rpc GetRepositoryCommitBySequenceId(GetRepositoryCommitBySequenceIdRequest) returns (GetRepositoryCommitBySequenceIdResponse);
+  // ListRepositoryDraftCommits lists draft commits in a repository.
```

Comment:
I am guessing that we will use `GetRepositoryCommitByReference` as opposed to a dedicated rpc for draft (`GetRepositoryCommitByDraft`)?